tests/lowlevel/test_simplify.py

```diff
@@ -0,0 +1,44 @@
+from msdsl import AnalogSignal, distribute_mult
```

Comment:
This is a new test that makes sure that the negation-related fix doesn't break anything in terms of how mixed-signal expressions can be simplified.